src/NavStyles.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+/**
+ *  Styles for the nav component.
+ *
+ * @see https://getbootstrap.com/docs/5.3/components/navs-tabs/#horizontal-alignment
+ */
+enum NavStyles: string
```

Comment:
We usually have enums named in singular, such as `DropdownAlignment`.

```suggestion
enum NavStyle: string
```

Reply:
done.